.gitignore

```diff
@@ -8,3 +8,4 @@ vendor/
 composer.lock
 doctrine-common-*.tar
 doctrine-common-*.tar.gz
+humbuglog.txt
```

Comment:
we have an ignored ``reports`` folder for tool reports. Please put the file in it instead of ignoring a new root file